alethzero/MainWin.cpp

```diff
@@ -1878,7 +1879,7 @@ void Main::on_dumpBlockState_triggered()
 		{
 			f << "{" << endl;
 //			js::mObject s;
-			State state = ethereum()->state(h);
+			State state = ethereum()->block(h).state();
```

Comment:
not tested this, this was just a quick fix to make it compile, please review carefully.